modules/networking/private_dns_resolvers_inbound_endpoints/variables.tf

```
@@ -0,0 +1,23 @@
+variable "global_settings" {
+  description = "Global settings object (see module README.md)"
+}
+variable "client_config" {
+  description = "Client configuration object (see module README.md)."
+}
+
+variable "settings" {
+}
+
+variable "location" {
+}
+
+variable "private_dns_resolver_id" {
+}
+variable "subnet_ids" {
+  type = list(string)
+}
+
```

You need to add a variable tags to get the base tags from the private_dns_resolver when inherit tags is set to true

```suggestion
variable "tags" {
  default = {}
}
```